CMakeLists.txt

```diff
@@ -0,0 +1,115 @@
+cmake_minimum_required(VERSION 3.0)
+
+project(KLEE VERSION 1.2.0)
+set(PACKAGE_STRING "\"KLEE ${KLEE_VERSION}\"")
+set(PACKAGE_URL "\"https://klee.github.io\"")
+set(KLEE_CMAKE_BUILD 1)
+set(RUNTIME_CONFIGURATION "\"\"")
+
+include(GNUInstallDirs)
+include(CheckIncludeFile)
+include(CheckIncludeFileCXX)
+include(CheckFunctionExists)
+include(CheckPrototypeDefinition)
+
+find_package(LLVM REQUIRED CONFIG)
+message(STATUS "Found LLVM ${LLVM_PACKAGE_VERSION}")
+message(STATUS "Using LLVMConfig.cmake in: ${LLVM_DIR}")
+
+add_definitions(${LLVM_DEFINITIONS})
+include_directories(${LLVM_INCLUDE_DIRS})
+link_directories(${LLVM_LIBRARY_DIRS})
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-rtti")
+#set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+
+add_definitions(-D_GNU_SOURCE
+	-DKLEE_DIR=\"${CMAKE_BINARY_DIR}\"
+	-DKLEE_INSTALL_BIN_DIR=\"${CMAKE_INSTALL_FULL_BINDIR}\"
+	-DKLEE_INSTALL_RUNTIME_DIR=\"${CMAKE_INSTALL_FULL_LIBDIR}/klee/runtime\")
+
+find_package(ZLIB REQUIRED)
+if (${ZLIB_FOUND})
+	set(HAVE_ZLIB_H 1)
```

Comment:
Minor nit. Please use an indentation of 2 spaces consistently through all the CMake files.